### PR TITLE
🚚 update: Add 131st CatchUp summary

### PR DESCRIPTION
**Pull request by HarshKapadia2**
So sorry for the delay in posting this summary. My parents had come to visit me and I had to move to another city, which is why this summary is more than two weeks late. Sorry!

Please review and merge this summary. (Do squash all commits before merging!)

Thank you!

### CODE REVIEW

**Comment by HarshKapadia2** [comment]
Do let me know if there are any changes or if I have forgotten anything. I'll make the necessary corrections.

**Comment by tusharnankani** [comment]
> So sorry for the delay in posting this summary. My parents had come to visit me and I had to move to another city, which is why this summary is more than two weeks late. Sorry!

It is alright, don't apologize for this. Glad you took the time out to spend time with them.

File: summary/sessions/131/content.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Date: 13-05-2023
+
+Duration: 4 hrs 41 mins
+
+==== Topics Discussed
+
+* General introductions.
+* link:https://twitter.com/guyinthecape[Vatsal Patel^] talked about his learnings at his workplace and how he is currently working with different teams to understand how they are working.
+	** He told us how he has been practising not asking someone to give him direct answers, but to only review his work, give him feedback and point him in the correct direction.
+		*** A caveat to that is that if the person whom one asks for help is undress stress, they might just provide direct answers as they don't have the time to have an elaborate discussion, so asking correct questions is important to get exactly what one wants. The responsibility of getting the type of mentorship one wants lies on oneself and cannot be entirely offloaded on someone else.
```
Comment (tusharnankani) [inline summary/sessions/131/content.adoc:10]: > undress stress

Interesting.

Is it under stress?

Comment (tusharnankani) [reply summary/sessions/131/content.adoc:10]:
```suggestion
		*** If someone is very busy and stressed, they may give direct answers when asked for help, without having time for a detailed discussion. So, it's important to ask the right questions to get the specific information one needs. It is our own responsibility to seek the type of mentorship we want and not entirely rely on someone else to provide it.
```

Complicated sentence, can we simply it to this?

**Review by tusharnankani — COMMENTED** [review]
Rest looks good.

File: summary/sessions/131/content.adoc (hunk 1 above)
Comment (HarshKapadia2) [reply summary/sessions/131/content.adoc:10]: Your suggestion makes a lot of sense! Thank you!

**Comment by HarshKapadia2** [comment]
Thank you for the review, Tushar! I'm squashing and merging this PR.

**Pull request merged**